staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate.go

```diff
@@ -53,7 +53,9 @@ type AnnotateOptions struct {
 	// Common user flags
 	overwrite       bool
 	local           bool
-	dryrun          bool
+	dryRunClient    bool
+	dryRunServer    bool
+	dryRunVerifier  *cmdutil.DryRunVerifier
```

Comment:
It doesn't hurt to keep the `dryRunVerifier` even though it could probably be thrown away now.